src/main/java/org/mitre/synthea/helpers/Utilities.java

```diff
@@ -18,11 +21,15 @@
 import java.util.function.BiConsumer;
 import java.util.function.Consumer;
 
+import org.hl7.fhir.r4.model.Bundle;
+import org.hl7.fhir.r4.model.Patient;
+
 import org.mitre.synthea.engine.Logic;
 import org.mitre.synthea.engine.State;
 import org.mitre.synthea.world.concepts.HealthRecord.Code;
 
 public class Utilities {
+  private static final FhirContext ctx = FhirContext.forR4();
```

Comment:
Are this & the other imports in this file actually needed? It's private and I don't see any other changes below this in the file